packages/studio/styled-system/types/conditions.d.ts

```diff
@@ -2,118 +2,118 @@
+	/** `@media (prefers-reduced-motion: reduce)` */"_motionReduce": string
+	/** `@media (prefers-reduced-motion: no-preference)` */"_motionSafe": string
+	/** `@media print` */"_print": string
+	/** `@media (orientation: landscape)` */"_landscape": string
+	/** `@media (orientation: portrait)` */"_portrait": string
+	/** ` &.dark, .dark &` */"_dark": string
+	/** ` &.light, .light &` */"_light": string
+	/** `@media (prefers-color-scheme: dark)` */"_osDark": string
+	/** `@media (prefers-color-scheme: light)` */"_osLight": string
+	/** `@media (forced-colors: active)` */"_highContrast": string
+	/** `@media (prefers-contrast: less)` */"_lessContrast": string
+	/** `@media (prefers-contrast: more)` */"_moreContrast": string
+	/** `[dir=ltr] &` */"_ltr": string
+	/** `[dir=rtl] &` */"_rtl": string
+	/** `&::-webkit-scrollbar` */"_scrollbar": string
+	/** `&::-webkit-scrollbar-thumb` */"_scrollbarThumb": string
+	/** `&::-webkit-scrollbar-track` */"_scrollbarTrack": string
+	/** `&[data-orientation=horizontal]` */"_horizontal": string
+	/** `&[data-orientation=vertical]` */"_vertical": string
+	/** `@media screen and (min-width: 640px)` */"sm": string
+	/** `@media screen and (min-width: 640px) and (max-width: 47.996875em)` */"smOnly": string
+	/** `@media screen and (max-width: 47.996875em)` */"smDown": string
+	/** `@media screen and (min-width: 768px)` */"md": string
+	/** `@media screen and (min-width: 768px) and (max-width: 63.996875em)` */"mdOnly": string
+	/** `@media screen and (max-width: 63.996875em)` */"mdDown": string
+	/** `@media screen and (min-width: 1024px)` */"lg": string
+	/** `@media screen and (min-width: 1024px) and (max-width: 79.996875em)` */"lgOnly": string
+	/** `@media screen and (max-width: 79.996875em)` */"lgDown": string
+	/** `@media screen and (min-width: 1280px)` */"xl": string
+	/** `@media screen and (min-width: 1280px) and (max-width: 95.996875em)` */"xlOnly": string
+	/** `@media screen and (max-width: 95.996875em)` */"xlDown": string
+	/** `@media screen and (min-width: 1536px)` */"2xl": string
+	/** `@media screen and (min-width: 1536px)` */"2xlOnly": string
+	/** `@media screen and (min-width: 640px) and (max-width: 63.996875em)` */"smToMd": string
+	/** `@media screen and (min-width: 640px) and (max-width: 79.996875em)` */"smToLg": string
+	/** `@media screen and (min-width: 640px) and (max-width: 95.996875em)` */"smToXl": string
+	/** `@media screen and (min-width: 640px)` */"smTo2xl": string
+	/** `@media screen and (min-width: 768px) and (max-width: 79.996875em)` */"mdToLg": string
+	/** `@media screen and (min-width: 768px) and (max-width: 95.996875em)` */"mdToXl": string
+	/** `@media screen and (min-width: 768px)` */"mdTo2xl": string
+	/** `@media screen and (min-width: 1024px) and (max-width: 95.996875em)` */"lgToXl": string
+	/** `@media screen and (min-width: 1024px)` */"lgTo2xl": string
+	/** `@media screen and (min-width: 1280px)` */"xlTo2xl": string
+	/** `undefined` */"base": string
```

Comment:
This looks great! Can we remove this undefined one from the base key?

Reply:
<img width="1291" alt="Screenshot 2023-09-03 at 10 49 56" src="https://github.com/chakra-ui/panda/assets/47224540/92ba871c-4b2f-4ff4-a5c4-214f8c7a5201">
Added this tiny bit of documentation instead

which made me realize we don't mention `base` anywhere in the docs, outside of examples